frontend/src/pages/FormStaffPage.vue

```diff
@@ -425,18 +342,17 @@ const updateClientType = (value: CodeNameType | undefined) => {
 
 const validation = reactive<Record<string, boolean>>({});
 
-const revalidateBus = useEventBus<void>("revalidate-bus");
-
 const goToStep = (index: number, skipCheck: boolean = false) => {
   if (skipCheck || (index <= currentTab.value && checkStepValidity(index)))
     currentTab.value = index;
   else notificationBus.emit({ fieldId: "missing.info", errorMsg: "" });
-  revalidateBus.emit();
+  revalidateBus.emit(progressData[currentTab.value].fields);
```

Comment:
Again, it seems like progressData[currentTab.value].fields is undefined. Is it intended?

Reply:
Yes it was intentional to pass undefined, but I'm removing the parameter as it has the same effect